### ISSUE TITLE
MouseEvent-prototype-offsetX-offsetY.html seems to depend on non-spec-defined behavior

### ISSUE THREAD

**Issue by bzbarsky** [web-platform-tests/wpt#15089]
The test is testing things like the offsetTop of an inline, but this is not actually defined in the CSS specification.  Specifically, the relevant text in CSS 2.1, last I checked, is:

>      10.6 Calculating heights and margins
>        10.6.1 Inline, non-replaced elements
>          The 'height' property does not apply. The height of the content area
>          should be based on the font, but this specification does not specify how.

and I'm not aware of any later CSS specs that define the behavior.

Since the offsetTop depends on the interaction of the line-height and the heights of the inlines, it's not specified either.

The test should probably use a `display: block` target element instead.

@rniwa, thoughts?  It looks like you wrote this test.

**Comment by bzbarsky** [comment]
Actually, we could probably stay with an inline if we set `line-height: 1; font-family: Ahem` on the `my-host` element.  That should eliminate the browser rendering differences in terms of inline layout.

**Comment by rniwa** [comment]
Yeah, setting `line-height` and `font-family` sounds great to me. I wanted to add testing for inline elements but didn't realize it wasn't specified at all. I guess we could use `display:block` if we wanted to avoid depending on any unspec'ed behavior.

**Comment by bzbarsky** [comment]
No, browsers are still not interoperable even with the Ahem/line-height thing, looks like...
